tests/attestation/attestation.bicepparam

```diff
@@ -0,0 +1,12 @@
+using './attestation.bicep'
+
+// Image info
+param registry='confidentialsidecars.azurecr.io'
+param tag=''
+
+// Deployment info
+param location='westeurope'
```

Comment:
Do you think we should do a matrix for location in the step and fill this value in? It would mimic the current examples repo and allow us to test in the different regions in this repo instead of the examples (if we're trying to remove the examples repo).

Reply:
I think the purpose here less about testing ACI which is (partially) the use case of the examples repo, and more about testing the sidecars themselves, so I don't think it's too important to test different regions, except for maybe having `eastus2euap` since that often will get new features first which might be good to know about (e.g. the new kernel versions)

Reply:
What I've settled on is having a separate workflow file which runs regularly and on demand to test the images in canary, I don't think failures in canary should block PRs